modules/nw-ingress-converting-http-header-case.adoc

```diff
@@ -31,6 +31,23 @@ As a cluster administrator, you can convert the HTTP header case by entering the
 $ oc -n openshift-ingress-operator patch ingresscontrollers/default --type=merge --patch='{"spec":{"httpHeaders":{"headerNameCaseAdjustments":["Host"]}}}'
 ----
 +
+. Create a `Route` resource YAML file so that the annotation can be applied to the application.
++
+.Example of a route named `my-application`
+[source,yaml]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+ name: my-application
+ namespace: default
```

Comment:
namespace: my-application
@dfitzmau  Kindly change the namespace from default to  my-application

Reply:
Thanks, Melvin. Updated.